build/release.mk

```diff
@@ -33,7 +33,6 @@ package:
 
 	@# Resource directories
 	mkdir -p $(DIST_PATH)/config
-	cp -L config/README.md $(DIST_PATH)/config
```

Comment:
Removed `README.md` as it /seemed/ unnecessary: https://github.com/mattermost/mattermost-server/tree/master/config.

Opinions welcome, but it's pretty weird to have that page without a `config.json` present in the source to explain it. Plus it will eventually go away with the database config. 0/5.

Reply:
Need to restore this copy.